src/cli/node/json_rpc_client.rs

```diff
@@ -247,6 +249,79 @@ pub fn get_utxo_info(
     Ok(())
 }
 
+#[allow(clippy::cast_possible_wrap)]
+pub fn get_supply_info(addr: SocketAddr) -> Result<(), failure::Error> {
+    let mut stream = start_client(addr)?;
+
+    let request = r#"{"jsonrpc": "2.0","method": "getSupplyInfo", "id": "1"}"#;
+    let response = send_request(&mut stream, &request)?;
+    let supply_info = parse_response::<SupplyInfo>(&response)?;
+
+    log::info!("{:?}", supply_info);
+
+    println!(
+        "\nSupply info at {} (epoch {}):\n",
+        pretty_print(supply_info.current_time as i64, 0),
+        supply_info.epoch
+    );
+
+    let block_rewards_wit =
+        Wit::wits_and_nanowits(Wit::from_nanowits(supply_info.blocks_minted_reward)).0;
+    let block_rewards_missing_wit =
+        Wit::wits_and_nanowits(Wit::from_nanowits(supply_info.blocks_missing_reward)).0;
+    let collateralized_data_requests_total_wit =
+        Wit::wits_and_nanowits(Wit::from_nanowits(supply_info.collateral_locked)).0;
+    let current_supply = Wit::wits_and_nanowits(Wit::from_nanowits(
+        supply_info.current_unlocked_supply + supply_info.collateral_locked,
+    ))
+    .0;
+    let locked_supply =
+        Wit::wits_and_nanowits(Wit::from_nanowits(supply_info.current_locked_supply)).0;
+    let total_supply = Wit::wits_and_nanowits(Wit::from_nanowits(
+        supply_info.total_supply - supply_info.blocks_missing_reward,
+    ))
+    .0;
+
+    let mut supply_table = Table::new();
+    supply_table.set_format(*prettytable::format::consts::FORMAT_NO_BORDER_LINE_SEPARATOR);
+    supply_table.set_titles(row!["Supply type", "Amount", "Total WITs"]);
+    supply_table.add_row(row![
+        "Blocks and WIT minted".to_string(),
+        supply_info.blocks_minted.to_formatted_string(&Locale::en),
+        block_rewards_wit.to_formatted_string(&Locale::en)
+    ]);
+    supply_table.add_row(row![
+        "Blocks and WIT missing".to_string(),
+        supply_info.blocks_missing.to_formatted_string(&Locale::en),
+        block_rewards_missing_wit.to_formatted_string(&Locale::en)
+    ]);
+    supply_table.add_row(row![
+        "Collateralized data requests".to_string(),
+        supply_info
+            .collateralized_data_requests
+            .to_formatted_string(&Locale::en),
+        collateralized_data_requests_total_wit.to_formatted_string(&Locale::en)
+    ]);
+    supply_table.add_row(row![
+        "Current supply".to_string(),
```

Comment:
Does `current` include `locked` or not?
- If included, I'd rename it to `Total`, but I'm also sort of fine with `Current`
- If not, I'd rename it to `Circulating`

Reply:
It does not, so circulating it is.